src/genbenchQC/cli.py

```
@@ -0,0 +1,77 @@
+from typing import List, Optional
+
+from genbenchQC.evaluate_classes import run as run_evaluate_classes
+from genbenchQC.evaluate_splits import run as run_evaluate_splits
+
+app = typer.Typer(no_args_is_help=True)
+
+@app.command()
+def evaluate_classes(
+    input: List[str] = typer.Option(..., help="Input file(s)."),
+    format: str = typer.Option(..., help="Input format: fasta, csv, csv.gz, tsv, tsv.gz"),
+    sequence_column: List[str] = typer.Option(['sequence'], help="One or more sequence column names for CSV/TSV inputs."),
```

Ruff (0.14.8)

Do not perform function call `typer.Option` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)